app/views/groups/edit.html.erb

```diff
@@ -24,20 +24,31 @@
     <tr>
       <td><%= user.full_name %></td>
       <td><%= user.email %></td>
-      <td>
-        <a href="#" class="btn btn-sm btn-warning text-light"><i class="bi bi-chat-fill"></i></a>
-        <a href="#" class="btn btn-sm btn-primary"><i class="bi bi-person-dash-fill"></i></a>
-      </td>
+      <% if not user.admin_in?(@group) %>
+        <td>
+          <a href="#" class="btn btn-sm btn-warning text-light"><i class="bi bi-chat-fill"></i></a>
+          <% if @user.admin_in?(@group) %>
```

Comment:
The group controller assures that the edit page is only accessible to admins, so this will be always true.

Reply:
Should be resolved by aaf2b72

Reply:
See also f4f01f2. Now CI is not failing anymore 👍🏻